docs/application/native/guides/connectivity/zigbee.md

```diff
@@ -0,0 +1,316 @@
+- [Form or Disable Zigbee Network Coordinator](#form-or-disable-zigbee-network-coordinator)
+- [Permit Join](#permit-join)
+- [Select Target Device](#select-target-device)
+
+Below is the list of supported hardware and chipset combinations.
+
+- Artik 5 (Inbuilt module)
+- Artik 7 (Inbuilt module)
+- Artik 10 (Inbuilt module)
+- CEL EM357 (External USB stick)
+
+This feature is supported in common profile only. Application should add below feature in its manifest file.
+
+    http://tizen.org/feature/network.zigbee
+
+> **Note**
+> - You can test the Zigbee functionality on a target device only. The [emulator](../../../tizen-studio/common-tools/emulator.md) does not support this feature.
+> - All callbacks in this module are called in the main loop context.
+
+## Prerequisites
+
+Below are the prerequisites for [Zigbee](../../api/mobile/latest/group__CAPI__NETWORK__ZIGBEE__MODULE.html) API:
+
+1. Add below platform level privilege to `tizen-manifest.xml` file for enabling and disabling Zigbee:
+
+   ```
+   <privileges>
+      <privilege>http://tizen.org/privilege/zigbee.admin</privilege>
+   </privileges>
+   ```
+
+2. Add below public level privilege to `tizen-manifest.xml` file for operations other than enabling and disabling Zigbee:
+
+   ```
+   <privileges>
+      <privilege>http://tizen.org/privilege/zigbee</privilege>
+   </privileges>
+   ```
+
+3. To use the functions and data types of the Zigbee API, include the below header file in your application:
+
+   ```
+   #include <zigbee.h>
+   ```
+
+4. Call the `zb_create()` function to create zigbee service handle:
+
+   ```
+   int zb_create(zb_zigbee_h *handle);
+   ```
+
+5. Call the `zb_destroy()` function to destroy zigbee service handle:
+
+   ```
+   int zb_destroy(zb_zigbee_h handle);
+   ```
+
+## Enable or Disable Zigbee Device
+
+1. To enable Zigbee, you should call `zb_enable()` API and wait for Zigbee state changed event `ZB_ZDP_ENABLE_EVENT`.
+
+   ```
+   static void _zb_event_cb(zb_nwk_addr addr16, zb_ieee_addr addr64, zb_event_e e,
+                            zb_event_data_h ev, void* user_data)
+   {
+       // Handle e for ZB_ZDP_ENABLE_EVENT
+   }
+
+   zb_zigbee_h enable_zigbee_device(void)
+   {
+       zb_zigbee_h handle;
+       int ret = ZIGBEE_ERROR_NONE;
+
+       ret = zb_create(&handle);
+       if (ZIGBEE_ERROR_NONE != ret) {
+           printf("zb_create() - FAILED!!!");
+           return NULL;
+       }
+
+       ret = zb_set_event_cb(handle, _zb_event_cb);
+       if (ZIGBEE_ERROR_NONE != ret) {
+           printf("zb_set_event_cb() - FAILED!!!");
+           return NULL;
+       }
+
+       ret = zb_enable(handle);
+       if (ZIGBEE_ERROR_NONE != ret) {
+           printf("zb_enable() - FAILED!!!");
+           return NULL;
+       }
+
+       return handle;
+   }
+   ```
+
+2. To disable Zigbee, you should call `zb_disable()` API and wait for Zigbee state changed event `ZB_ZDP_ENABLE_EVENT`.
```

Comment:
you must call..

Please use "must" to emphasis the action.

Reply:
done

Reply:
please review the files again, I have amended the requested changes and uploaded the patchset.